app/lib/foreman_bootdisk/scope/bootdisk.rb

```diff
@@ -18,6 +17,10 @@ def bootdisk_chain_url(mac = host.try(:mac), action = 'iPXE')
       def bootdisk_raise(*args)
         raise ::Foreman::Exception.new(*args)
       end
+
+      def template_name
```

Comment:
I believe this is a method that should be defined for the source, not the actual template scope.

@kamils-ironin: Do you remember?

Reply:
I think you are right, the template name is taken from the source

https://github.com/theforeman/foreman/blob/7b0e8889cbff82e6b5caf1e09752f17d0dbe3ffc/lib/foreman/renderer/scope/base.rb#L26

Reply:
I need to do it this way because bootdisk actually also renders PXE Linux templates using the default scopes.